support-payment-api/src/main/scala/backend/AmazonPayBackend.scala

```diff
@@ -124,6 +129,14 @@ class AmazonPayBackend(cloudWatchService: CloudWatchService,
     ophanService.submitAcquisition(acquisition)
       .bimap(BackendError.fromOphanError, _ => ())
 
+  private def submitAcquisitionToBigQuery(
+    acquisition: AmazonPayAcquisition,
+    contributionData: ContributionData
+  ): EitherT[Future, BackendError, Unit] =
+    EitherT.fromEither[Future](bigQueryService.tableInsertRow(
```

Comment:
Any idea if we need to worry about this being a blocking call?
Currently we return a response to the client after the `trackContribution` tasks have finished, even though their result does not affect the response. Which is perhaps not ideal

Reply:
This is a good point, I wasn't too worried about it in a lambda but on an EC2 box we probably need to be a bit more careful. I've wrapped the call in a blocking future as suggested here:
https://docs.scala-lang.org/overviews/core/futures.html#blocking-inside-a-future

Reply:
thanks!